pkg/controller/admissionchecks/multikueue/multikueuecluster.go

```diff
@@ -98,30 +122,65 @@ func (rc *remoteClient) setConfig(watchCtx context.Context, kubeconfig []byte) e
 	}
 	rc.client = remoteClient
 
-	newWatcher, err := rc.client.Watch(watchCtx, &kueue.WorkloadList{})
+	err = rc.startWatcher(watchCtx, kueue.GroupVersion.WithKind("Workload").GroupKind().String(), &workloadKueueWatcher{})
```

Comment:
Add the comment here as well to use the origin label when introduced

Reply:
done